docs/tools/kit/index.md

```diff
@@ -377,11 +377,12 @@ function RandomValues() {
 
 #### Notes:
 
-* Randomness is generated using Flow’s **on-chain `revertibleRandom`**, producing pseudorandom values tied to block and transaction execution.
-* The value returned for identical calls within the same block will be identical.
-* This hook is designed for simple use cases that **don’t require a [commit-reveal scheme]**, such as randomized UIs.
-* For **more advanced use cases** that **do** require a [commit-reveal scheme], Flow provides built-in support for that pattern.
-* Values are **deterministic**: if a transaction fails and retries, the same random value will be regenerated.
+* Randomness is generated using Flow’s **on-chain `revertibleRandom`**, producing pseudorandom values tied to block and script execution.
+* Values are **deterministic**: The values returned for identical calls within the same block will be identical.
+* If `count ` is larger than one, the returned values are disctinct.
+* This hook is designed for simple use cases that don't require unpredictability, such as randomized UIs.
+  Since the hook uses script executions on existing blocks, Flow's randon source is already public and the randoms are predictable. 
+* For **more advanced use cases** that **do** require on-chain randomness logic via transactions, Flow provides built-in support using Cadence's `revertibleRandom` and [commit-reveal scheme].
```

Comment:
The more advanced cases are the ones with the logic fully on-chain via transactions (not scripts). This is where randomness is unpredictable, and reversibility can or cannot be an issue, so I added the mention of transactions here. Both "reversibleRandom" and "commit-reveal" work in that case depending on the app.